lib/surrealist/copier.rb

```diff
@@ -44,6 +39,43 @@ def copy_hash(hash, wrapper: {})
         end
       end
 
+      # Wraps schema into a root key if `include_root` is passed to Surrealist.
+      #
+      # @param [Hash] schema schema hash.
+      # @param [String] klass name of the class where schema is defined.
+      # @param [Boolean] camelize optional camelize argument.
+      #
+      # @return [Hash] a hash with schema wrapped inside a root key.
+      def wrap_schema_into_root(schema:, klass:, camelize:)
```

Comment:
Only thing I can see is this too consider. Only would be using one attr from the Carrier here but not sure if consistency is more important or not.

Reply:
Yep, you are right, i've missed that. I guess consistency is better.